### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by pygaurav**
These are the changes for the new req.

Please find attached images.

512px-Python_icon_(black_and_white) svg

### CODE REVIEW

File: src/components/AdminContent/AdminContent.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const AdminContent = () => {
+    return (
+        <div>
+            
+        </div>
+    )
+}
```
Comment (SuyashGupta8) [inline src/components/AdminContent/AdminContent.js:10]: whats this:???

Comment (pygaurav) [reply src/components/AdminContent/AdminContent.js:10]: F*** off

**Review by SuyashGupta8 — CHANGES_REQUESTED** [review]
?F*

**Review by SuyashGupta8 — APPROVED** [review]
Good buk

**Pull request merged**